Improve run.sh script

CermakM opened the pull request:
- previously not working when running locally without credentials
- provide information about required credentials

- simple command line usage

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   README.rst
modified:   run.sh

Commit:
Improve run.sh script

- previously not working when running locally without credentials
- provide information about required credentials

- simple command line usage

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   README.rst
modified:   run.sh

CermakM commented:
@fridex @goern Could you have a look, please? I am not allowed to assign reviewers.

fridex commented on run.sh:
```diff
@@ -1,19 +1,108 @@
+	[ -z "$THOTH_CEPH_HOST" ]          && echo "ERROR: THOTH_CEPH_HOST must be set"  		 && exit 1
+	[ -z "$THOTH_CEPH_BUCKET" ]		   && echo "ERROR: THOTH_CEPH_BUCKET must be set" 	  	 && exit 1
+	[ -z "$THOTH_CEPH_BUCKET_PREFIX" ] && echo "ERROR: THOTH_CEPH_BUCKET_PREFIX must be set" && exit 1
+	[ -z "$JANUSGRAPH_SERVICE_HOST" ]  && echo "ERROR: JANUSGRAPH_SERVICE_HOST must be set"  && exit 1
```
You can provide something like `die`:

https://github.com/thoth-station/janusgraph-thoth-config/blob/dc7244072cd9365afd9e3d93ff28f6287dbf4028/local.sh#L18-L21

CermakM replied:
That's neat... will do, thanks!

fridex commented on run.sh:
```diff
@@ -1,19 +1,108 @@
+	echo -e "\033[33;1mINFO: Installing...\033[0m"
+	echo -e "\033[33;1mNOTE: Grab a coffee, this will take a while.\033[0m"
+	if [ "${VERBOSE}" = "true" ]; then
+		/usr/bin/env bash ./install.sh || exit $?
```
At the beginning, there is `set -e` so `exit $?` shouldn't be needed. The same for `exit` below with PIPESTATUS.

fridex reviewed (commented):
Ideally we could maintain profiles in `gopass` with all the secrets, configuration options and load configuration from there as in the eval statement. This could be especially useful when talking to multiple deployments (e.g. talk to `test`, `stage`, `local`). But that is just an idea for future improvements.

Commit:
Changes based on review and pretty print

- use common `die` function instead of exits
- get rid of unnecessary exit status calls

- pretty print credentials in help output

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   run.sh

fridex commented on run.sh:
```diff
@@ -1,19 +1,117 @@
-	echo "You do not have access. Please check your accesibility rights"
 fi
+
+# pipenv run jupyter notebook --notebook-dir=notebooks --NotebookApp.token=''
```
I think this commented out line is a left-over.

CermakM replied:
Oh ye, I just noticed it, too. Force pushing

fridex reviewed (approved):
Nice, haven't tried it. I trust you.

CermakM commented:
Thanks!
